apis/kubedb/v1/types.go

```diff
@@ -204,6 +204,14 @@ type SystemUserSecretsSpec struct {
 }
 
 type SecretReference struct {
+	// +optional
+	// Two possible groups: "", virtual-secrets.dev
+	Group string `json:"group,omitempty"`
+
+	// +optional
+	// SecretSource references the secret manager used for virtual secret
+	SecretSource string `json:"secretSource,omitempty"`
```

Comment:
Why the secretSource is needed here? 
The `VirtualScret` has `.spec.secretSource` field already

Reply:
This is like specifying `storageClassName` in `volumeClaimTemplate`. When provisioner creates new virtual secret, it needs to know which source will be used to store it. We could implement a default source class at some point.